### PR TITLE
add ability for match to upload arbitrary files

### PR DESCRIPTION
**Pull request by addbrick**

### CODE REVIEW

File: match/lib/match/options.rb [hunk 1]
```diff
@@ -16,6 +16,14 @@ def self.available_options
 
       [
         # main
+        FastlaneCore::ConfigItem.new(key: :sync_encrypted_arbitrary_directory,
+                                     env_name: "MATCH_sync_encrypted_arbitrary_directory",
```
Comment (saim80) [inline match/lib/match/options.rb:20]: For consistency, it'd be nicer if uppercased.

File: match/lib/match/runner.rb [hunk 2]
```diff
@@ -106,17 +132,27 @@ def run(params)
         storage.save_changes!(files_to_commit: self.files_to_commit)
       end
 
-      # Print a summary table for each app_identifier
-      app_identifiers.each do |app_identifier|
-        TablePrinter.print_summary(app_identifier: app_identifier, type: params[:type], platform: params[:platform])
+      if sync_encrypted_arbitrary_directory_path
+        puts "Encrypted and uploaded directory :: #{sync_encrypted_arbitrary_directory_path} ::"
```
Comment (saim80) [inline match/lib/match/runner.rb:137]: We should use Fastlane's logging system to print this out.

**Review by saim80 — APPROVED** [review]
The changes are fine by me, though I doubt if `Fastlane` maintainer would accept this change. The capability added in this pr is beyond the defined scope of `Match`. It would be more appropriate to make this as a Fastlane Action add-on and distribute publicly.

**Comment by addbrick** [comment]
Thanks @saim80, I agree it does feel out of scope for match. But it is just way easier to piggy back off the existing runner. I am curious on what they will think

**Comment by addbrick** [comment]
Turns out this is not necessary, I was able to accomplish the same thing using the built in commands in a lane

**Pull request closed without merging**